src/resin-event-log.js

```diff
@@ -86,6 +86,7 @@ var EVENTS = {
 	releaseTag: [ 'set', 'create', 'edit', 'delete' ],
 	billing: [ 'paymentInfoUpdate', 'planChange', 'invoiceDownload' ],
 	onboarding: [ 'stepClick', 'whatNextItemClick'],
+	gettingStartedGuide: ['showModalGroup', 'hideModalGroup', 'modalSkipClick', 'modalOpenGuideClick'],
```

Comment:
```suggestion
	gettingStartedGuide: ['modalShow', 'modalHide', 'modalSkip', 'modalGuideOpen'],
```

Just so it matches the naming of other methods better and is less verbose.